src/components/Wallet/index.jsx

```diff
@@ -562,6 +562,8 @@ CreateWallet.propTypes = {
     fetchUTXOsErrors: PropTypes.number,
     nodes: PropTypes.shape({}),
   }).isRequired,
+  confirmedBalance: PropTypes.oneOfType([PropTypes.number, PropTypes.string])
```

Comment:
I think this is because of how we need to use BigNumbers. It is almost always being passed through as a string actually I think (you can see this in the state), but I've been putting both to catch edge cases because of the complexity of state management and these reducers.